pkg/eventbus/mqtt/server.go

```diff
@@ -0,0 +1,106 @@
+package mqtt
+
+import (
+	"encoding/base64"
+	"fmt"
+	"strings"
+
+	"github.com/256dpi/gomqtt/broker"
+	"github.com/256dpi/gomqtt/packet"
+	"github.com/256dpi/gomqtt/topic"
+	"github.com/256dpi/gomqtt/transport"
+
+	"github.com/kubeedge/kubeedge/beehive/pkg/common/log"
+	"github.com/kubeedge/kubeedge/beehive/pkg/core"
+	"github.com/kubeedge/kubeedge/beehive/pkg/core/model"
+)
+
```

Comment:
Please add comment on top of structures and functions.
Applicable in all similar places.